guillotina/api/content.py

```diff
@@ -178,11 +178,18 @@ class DefaultPOST(Service):
 
         user = get_authenticated_user_id(self.request)
 
+        options = {
+            'id': new_id,
+            'creators': (user,),
+            'contributors': (user,)
+        }
+        if 'uid' in data:
```

Comment:
Why we need to be able to define uid on POST creation?

Reply:
In running migrations from plone, you need to match UIDs so linking works properly